app/views/find/result_filters/_visa_filter.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="govuk-checkboxes__item">
       <%= form.check_box(
             :can_sponsor_visa,
-            { checked: @filters_view.visa_checked?, class: "govuk-checkboxes__input" },
+            { checked: !@filters_view.visa_checked?, class: "govuk-checkboxes__input" },
```

Comment:
The can sponsor visa checked is expexted a negation of `visa_checked?`. Need to understand a little, can you clarify, please?

Reply:
It's this: https://github.com/DFE-Digital/publish-teacher-training/pull/3906#pullrequestreview-1704003244

<img width="1019" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/7c4b3758-c05e-4e83-b2fa-ff0790e11c36">

Does that make sense?

Reply:
Sorry I should had phrase better. the code `checked: !@filters_view.visa_checked?` is confusing for me, because the #visa_checked? method returns true we can expect the filter to be checked, but the code is showing the inverse, if `#visa_checked?` returns true the checked option will be false.

Reply:
I've just given @avinhurry a couple of comments on this filter - it appears it's showing results the wrong way round at present. That is to say:
- If I say `Yes` to the Q: Do you need visa sponsorship, I see all courses, including those that don't sponsor visas
- If I say `no` to the Q: Do you need visa sponsorship, I see limited courses, only ones that sponsor visas.